lib/game.dart

```diff
@@ -80,6 +80,9 @@ abstract class BaseGame extends Game {
   /// By default, this calls the first time resize for every component, so don't forget to call super.preAdd when overriding.
   @mustCallSuper
   void preAdd(Component c) {
+    if (debugMode() && c is PositionComponent) {
```

Comment:
add to the docs of debugMode of the game class the fact that it cannot be changed in runtime